templates/ingest/{{ cookiecutter.module }}/config/retriever.yaml

```diff
@@ -0,0 +1,45 @@
+classname: tsdat.io.retrievers.DefaultRetriever
+readers:
+  .*:
+    {% if cookiecutter.use_custom_data_reader == "no" -%}
+    classname: tsdat.io.readers.CSVReader
+    parameters:
+      read_csv_kwargs:
+        # This treats the first column (time) as a data variable, which helps for when
+        # the pandas DataFrame is converted to an xarray Dataset
+        index_col: false
+
+        # This tells pandas to look at row index 2 (0-based) for the header
+        header: 2
+    {% else -%}
+    classname: pipelines.{{ cookiecutter.module }}.readers.CustomDataReader
+    {% endif %}
+coords:
+  # DEVELOPER: Specify the coords/data_vars that should be retrieved from any inputs
+  time:
+    # Mapping of regex pattern (matching input key/file) to input name & converter(s) to
+    # run. The default is .*, which matches everything. Put the most specific patterns
+    # first because searching happens top -> down and stops at the first match.
+    .*:
```

Comment:
You don't worry about this for now, but I'd like to change the template so that users don't need to think about the regex here. This can be done later, for other active repositories.